app/components/header/header.component.html

```diff
@@ -106,6 +106,17 @@
             <a href="#"><img src="./assets/stash.png" height="40" width="48" class="add-repository-button" title="Stash"></a>
             <span class="toolbar-Text">Stash</span>
         </li>
+
+        <!--Add Upstream Remote Respository-->
+        <li class="toolbar-Icons" onclick="setUpstreamModal()">
+            <a href="#"><img src="./assets/addremote.png" height="40" width="48" class="add-repository-button" title="Set Upstream"></a>
```

Comment:
Recommend using font awesome instead of custom assets, much more lightweight and scalable. Didn't see a fork one but did see this icon. https://fontawesome.com/

Reply:
Great recommendation! I'll look into this